redirect/routes.php

```diff
@@ -496,10 +496,25 @@ class Route {
             'target'      => 'https://tum.sexy/pret/',
         ],
         'nix'          => [
-             'description' => 'GLÜHNIX',
+            'description' => 'GLÜHNIX',
             'target'      => 'https://www.facebook.com/events/807341503020074/',
         ],
-            
+        'eat-api' => [
+            'description' => 'eat-api output in a human readable format as well as the base url for eat-api calls',
+            'target'      => 'https://tum-dev.github.io/eat-api/',
+        ],
+        'all.eat-api'  => [
```

Comment:
I'm pretty sure this won't work. We don't have a wildcard certificate for sub-subdomains as @mammuth pointed out. You can use hyphens throughout however.